modules/local/annotate_gff.nf

```diff
@@ -7,20 +7,21 @@ process ANNONTATE_GFF {
     container 'quay.io/microbiome-informatics/genomes-pipeline.python3base:v1.1'
```

Comment:
We shouln't be using this container... we should use the bioconda python base one. I'll replace this one tomorrow